config/application.rb

```diff
@@ -18,6 +18,16 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
+    config.i18n.default_locale = :en
+    config.i18n.fallbacks = {
+      en_us: :en,
+      en_ca: :en,
+      en_uk: :en,
+      fr_lu: :fr,
+      fr_ca: :fr,
+      fr:    :en,
+      es:    :en,
```

Comment:
What's the difference between a default locale and fallbacks?  And why these fallbacks?

Reply:
Locale is for default when there are more than 1 existing translations, fallbacks are for when the translation doesn't exist and falls back to an existing one.